pkg/liqo-controller-manager/storageprovisioner/storageprovisioner_test.go

```diff
@@ -79,6 +80,10 @@ var _ = Describe("Test Storage Provisioner", func() {
 					ObjectMeta: metav1.ObjectMeta{
 						Name:      name,
 						Namespace: namespace,
+						UID: func() apitypes.UID {
+							uid, _ := uuid.NewUUID()
+							return apitypes.UID(uid.String())
+						}(),
```

Comment:
Could assign a fake value instead of generating a real UUID, which is even simpler.